README.md

```diff
@@ -54,6 +54,11 @@ $ S3_GW_PEERS_0_ADDRESS=grpcs://192.168.130.72:8080 \
   neofs-s3-gw
 ```
 
+## Restrictions
```

Comment:
We better have it in `docs/aws_s3_compat.md`, another section on metadata (like we have for ACLs) would be appropriate here.

Reply:
Make sense, missed this doc location. Updated